frontend/src/components/MenuBar/MenuBar.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { StyleSheet } from "react-native";
+import { useNavigation } from "@react-navigation/native";
+import { View, Text, TouchableHighlight } from "react-native";
```

Comment:
Do we need Text and TouchableHighlight here? I don't imagine we will use Text here and I have no clue what TouchableHighlight is.

Navigation is also imported and declared but never used. If you think we will use it later that is fine.